api/test/tests/controllers/meeting.test.js

```diff
@@ -224,33 +247,68 @@ describe("lib/controllers/meeting", () => {
-      const ownedMeeting = fakeMeeting({
-        owner_id: this.user._id,
-        name: "meeting 1",
-        date: new Date("05 October 2011 14:48 UTC"),
+      const includedMeetingOwner = await User.create(fakeUser());
+      const includedMeeting = fakeMeeting({
+        owner_id: includedMeetingOwner._id,
+        name: "participant meeting",
       });
-      const ownedMeeting2 = fakeMeeting({
-        owner_id: this.user._id,
-        name: "meeting 2",
-        date: new Date("01 January 1990 01:22 UTC"),
+
+      await Promise.all([
+      await Meeting.create(this.ownedMeeting),
+      await Meeting.create(this.ownedMeeting2), //we will limit before getting to this oldest meeting
+      await Meeting.create(this.ownedMeeting3), //we will skip this newest meeting for pagenation test
+      await Meeting.create(this.ownedMeeting4),
+      ])
+     
+      const includedRes = await Meeting.create(includedMeeting);
+
+      const participant = fakeParticipant({
+        meeting_id: includedRes._id,
+        email: this.user.email,
       });
-      const ownedMeeting3 = fakeMeeting({
-        owner_id: this.user._id,
-        name: "meeting 3",
-        date: new Date("05 October 2500 14:48 UTC"),
+
+      await Participant.create(participant);
+
+      const { data } = await client.get(`/meeting/?skip=1&limit=3`);
+
+      assert.strictEqual(data.meetings.length, 3); //only send user 3 because of pagination
+      assert.strictEqual(data.count, 5);           //should still be 5 total (including other pages)
+      assert.strictEqual(data.meetings[0].name, "participant meeting"); //Second newest meeting first
+      assert.strictEqual(data.meetings[2].name, "meeting 4"); //Second oldest meeting last
+      assert.exists(data.owners);
+    });
+
+    it("should filter meetings by name", async () => {
+      
+      await Promise.all([
+      Meeting.create(this.ownedMeeting),
+      Meeting.create(this.ownedMeeting2),
+      Meeting.create(this.ownedMeeting3),
+      Meeting.create(this.ownedMeeting4),
+      ]);
+
+      const filters = { owners: [], name: "meeting 1" };
+
+      const { data } = await client.get(`/meeting/?skip=0&limit=2`, {
+        params: filters,
       });
-      const ownedMeeting4 = fakeMeeting({
-        owner_id: this.user._id,
-        name: "meeting 4",
-        date: new Date("25 December 1995 01:22 UTC"),
+
+      assert.strictEqual(data.meetings.length, 1);
+      assert.strictEqual(data.meetings[0].name, "meeting 1");
+    });
+
+    it("should filter meetings by owner", async () => {
+      const includedMeetingOwner = await User.create(fakeUser());
+      const includedMeeting = fakeMeeting({
+        owner_id: includedMeetingOwner._id,
+        name: "participant meeting",
       });
 
-      const includedMeeting = fakeMeeting({ name: "participant meeting" });
+      await Promise.all([
+      await Meeting.create(this.ownedMeeting),
+      await Meeting.create(this.ownedMeeting2),
+      ])
```

Comment:
```suggestion
      await Promise.all([
        Meeting.create(this.ownedMeeting),
        Meeting.create(this.ownedMeeting2),
      ])
```
You can remove these `await`s here.

Reply:
whoops my bad